src/intersections/rayline.jl

```diff
@@ -14,16 +14,20 @@ function intersection(f, r1::Ray{N,T}, l1::Line{N,T}) where {N,T}
   a, b = r1(0), r1(1)
```

Comment:
@dorn-gerhard can we get the arguments of these functions renamed to something more descriptive first? Let's rename r1 to ray and l1 to line so that we don't get lost with all the r variables in the code meaning ranks.

Reply:
I totally agree, better to use `ray`, `seg` and `line`.